.github/workflows/k6tests-rg-deploy.yml

```
@@ -147,7 +148,7 @@ jobs:
           INIT_STEP_OUTPUT: ${{steps.terraform_init.outputs.init_step_stdout}}
           INIT_STEP_OUTCOME: ${{steps.terraform_init.outputs.init_step_outcome}}
           PLAN_STEP_EXITCODE: ${{steps.terraform_plan.outputs.plan_step_exitcode}}
-          PLAN_STEP_OUTCOME: ${{steps.terraform_plan.outputs.plan_step_outcome}}
+          PLAN_STEP_OUTCOME: ${{ steps.terraform_plan.outputs.plan_step_outcome != "" ? steps.terraform_plan.outputs.plan_step_outcome steps.terraform_partial_plan.outputs.plan_step_outcome}}
```

actionlint (1.7.7)

got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)